README.md

```
@@ -211,21 +211,21 @@ basecamp includes optional plugins for extended functionality. These are complem
-| `ws-cursor` | Discovers `.cursor/*.mdc` context files |
-| `ws-gpg-check` | Verifies GPG card before git commit/tag |
+| `bc-collab` | Collaborative discovery and planning for ideation and requirements |
+| `bc-eng` | Code review, PR workflows, testing patterns, Python/SQL development |
+| `bc-cursor` | Discovers `.cursor/*.mdc` context files |
+| `bc-gpg-check` | Verifies GPG card before git commit/tag |
 
 ### Installing Plugins
 
 From within a Claude Code session started via basecamp:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)